pom.xml

```diff
@@ -100,6 +100,7 @@
 		<!-- dependency versions -->
 		<jackson.version>2.6.5</jackson.version>
 		<junit.version>4.12</junit.version>
+		<jol.version>0.2</jol.version>
```

Comment:
@mbasmanova Is there a particular reason you used version 0.2? It seems the latest is 0.9.

Reply:
@stolstov Sergey, no particular reason. I used the same version as Presto does.